src/kOS/Module/kOSProcessor.cs

```diff
@@ -75,6 +75,19 @@ public string Tag
         [KSPField(isPersistant = true, guiActive = false)]
         public int MaxPartId = 100;
```

Comment:
I believe my experience has been that extra field info is ignored if there is no matching field on the C# class.  The same is true for unrecognized modules.  It is possible that the next save will not include the data, but that shouldn't be a problem if we remove the reference entirely.